### PR TITLE
feat: Overhaul localization for user feedback

### LINKED ISSUE #8166: Ability to customize the text in the user feedback form
Is it possible to edit the text on the user feedback form.

screen shot 2018-04-24 at 10 59 23 am

Specificially this text:
```
It looks like we're having some internal issues.
Our team has been notified. If you'd like to help, tell us what happened below.
```

### LINKED ISSUE #8760: Strange localization issue in user feedback dialog
This is concerning Czech localization, but I suspect others might have the same issue.

https://github.com/getsentry/sentry/blob/5a1da65a100a9033dbc7b93e4082570cdc477940/src/sentry/locale/cs/LC_MESSAGES/django.po#L954
image

The translation text is correct here and yet for some reason when the dialog is shown to the user, it looks like this. For some reason, only first half is translated, but a second part contained within `span` is kept original and even cut off.

Is this some known issue in the translation software you are using? It would probably help to strip down that `span` element there as it causing issues obviously.

### PR DESCRIPTION
**Pull request by dcramer**
- Remove unused tags in user feedback localization (discards existing translations)
- Support ?lang=[code] via locale middleware to force language on a single page load
- Correct debug page for embed QA
- Support customization of most strings

Fixes GH-8760
Fixes GH-8166

### CODE REVIEW

**Comment by dcramer** [comment]
Fixes GH-8760

**Comment by dcramer** [comment]
I'd be willing to immediately enable the label customization options assuming no one points out any serious negative consequences of them.

File: src/sentry/templates/sentry/debug/error-page-embed.html [hunk 1]
```diff
@@ -1,12 +1,6 @@
+{% load sentry_helpers %}
 <html>
 <body>
   <h2>Error Embed Debug</h2>
-
-  <script>
-  window.sentryConfig = {
```
Comment (dcramer) [inline src/sentry/templates/sentry/debug/error-page-embed.html]: turns out this was just straight up broken

File: src/sentry/web/frontend/error_page_embed.py [hunk 2]
```diff
@@ -171,22 +198,24 @@ def dispatch(self, request):
...
-                {
-                    'generic_error': six.text_type(GENERIC_ERROR),
-                    'form_error': six.text_type(FORM_ERROR),
-                    'sent_message': six.text_type(SENT_MESSAGE),
-                }
-            ),
+            'endpoint': mark_safe('*/' + json.dumps(request.build_absolute_uri()) + ';/*'),
+            'template': mark_safe('*/' + json.dumps(template) + ';/*'),
+            'strings': json.dumps_htmlsafe({
+                'generic_error': six.text_type(GENERIC_ERROR),
```
Comment (dcramer) [inline src/sentry/web/frontend/error_page_embed.py]: these would need customizable as well

**Comment by dcramer** [comment]
Actually the span changes might be a concern as they were intended to resolve mobile display issues.

**Review by mattrobenolt — APPROVED** [review]
Seems fine.

**Comment by dcramer** [comment]
screen shot 2018-07-10 at 2 08 34 pm

**Pull request merged**